web/docs/pick-a-tutorial.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Pick a Tutorial
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Congrats, you're now familiar with the basic concepts of Wasp! 🥳 
+
+It's time to build something cool and check the capabilities of Wasp in action. Pick one of the app tutorials (there will be more of them with time) and leave us your impressions on Discord [![Discord](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX)
```

Comment:
Aha it is visible on the left! I was thinking though that maybe we could list them here and then give a bit of explanation on which one would they choose. For example we could explain that excuses one is quick, takes x minutes and doesn't cover everything but is good to get fast overview, while TODO takes longer but explains things in more details an d covers more features of Wasp. Otherwise I don't see how they will be able to make the choice between the two.

Reply:
Valid point! Updated